### DISCUSSION TITLE
Can i add links in my profile's bio section?

### DISCUSSION THREAD

**Discussion by vikasganiga05** [EddieHubCommunity/LinkFree#1322] [Q&A] [answered]
Can I add links in my LinkFree profile bio section? I have not tried it myself. Anyone who tried or knows about it please let me know.

For example, I have the following user data.

```JSON
{
  "name": "Shivam Sharma",
  "type": "personal",
  "bio": "Member of <a href='https://github.com/EddieHubCommunity'>@EddieHubCommunity</a> | Full Stack Web Developer | Open Source & Cloud & DevOps Enthusiast.",
  "avatar": "https://github.com/shivam-sharma7.png",
  "links": ["..."]
}
```
The anchor tag will render on the profile page as a link or as a markup text?

**Comment by schmelto** [comment, marked as answer]
I've tested it and it will be displayed as string:

![image](https://user-images.githubusercontent.com/30869493/170070494-c538a7f1-a14a-43ae-a1c0-a6e36f63a619.png)

![image](https://user-images.githubusercontent.com/30869493/170070410-be18aedb-0c5d-40e7-aa2e-7a420f92fd16.png)

**Reply by vikasganiga05** [reply]
Thanks, @schmelto , For testing :pray: 
I think allowing links in the bio section can be a new feature. What are your thoughts on this Tom & other folks?

**Reply by schmelto** [reply]
> Thanks, @schmelto , For testing :pray: 
> 
> I think allowing links in the bio section can be a new feature. What are your thoughts on this Tom & other folks?

Like the idea maybe open an issue for this :)

**Reply by eddiejaoude** [reply]
Markdown is allowed in the bio so links can be added, check my profile
